ui/modal/info_modal.go

```diff
@@ -73,12 +81,23 @@ func (in *InfoModal) Dismiss() {
 }
 
 func (in *InfoModal) OnResume() {
+	in.textInputEditor.Editor.Focus()
 }
 
 func (in *InfoModal) OnDismiss() {
 
 }
 
+func (in *InfoModal) ShowTextEditor(showTextEditor bool) *InfoModal {
+	in.showEditor = showTextEditor
+	return in
+}
+
+func (in *InfoModal) TextInputEditor(editor decredmaterial.Editor) *InfoModal {
+	in.textInputEditor = editor
+	return in
+}
+
```

Comment:
Kindly undo the changes to this modal.

We have the text input modal for this already.